docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+
+
+# Managing neuroscience projects with **datashuttle**
+*Create, validate and transfer standardised project folders*
+
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Maintaining a well-organised neuroscience project is hard. 
+
+Although everyone can appreciate the benefits of a tidy project
+folder, the practicalities of running an experiment often gets 
+in the way. Folder organisation 
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
+The best protection against such errors is automating the process
+through acquisition scripts—resulting in hours spent writing code 
+managing the naming and transfer of folders—a task entirely 
+unrelated to the central research goals.
+
+In our previous blog, we highlighted the benefits of data standardisation 
+for systems neuroscience, introducing the 
+[NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) 
+specification. 
+An immediate benefit of a widely-used standard is that the entire community
+can contribute to shared tools for project management.
+This means individual researchers don't waste time
+writing data-management code. 
+
+In this blog post we introduce 
+[**datashuttle**](https://datashuttle.neuroinformatics.dev/)—a 
+tool for the automated creation, 
+validation and transfer of projects organised to 
+the **NeuroBlueprint** standard. **datashutte** aims to
+drop into existing acquisition pipelines, reducing errors
+associated with manual folder creation and removing the need
+to write your down data-management code.
+
+Below we give a whistlestop tour of **datashuttles** and it's key
+features. More information is available at the 
+[datashutte](https://datashuttle.neuroinformatics.dev/) 
+website.
+
+## How **datashuttle** is used in an experiment
+
+```{image} /_static/blog_images/datashuttle/tutorial-1-example-file-tree-dark.png
+:align: center
+:class: only-dark
+:width: 400px
+```
+```{image} /_static/blog_images/datashuttle/tutorial-1-example-file-tree-light.png
+:align: center
+:class: only-light
+:width: 400px
+```
+<br>
+
+Imagine that you are starting a new experiment and have the first
```

Comment:
"during which you acquire both behavioural (`behav`) and electrophysiological (`ephys`) data" ?

Reply:
also, this sentence should probably go before the image showing the data.